### PR TITLE
Install script improvements.

### PR DESCRIPTION
**Pull request by Pysis868**
Added ability to persist custom local configuration through externally referenced batch file.
Example file is provided.
For local custom file, added to gitignore.
With this to be able to test faster, added variable for selecting tasks manually, and also when mysql connection information is explicitly given, skipping that page as well.
Changed all mysql calls to use connection string variable layer to aid in customization, also involded in the above mentioned.

From these additions, primitive parameters are also added for forcing prompt or none in case a condition is otherwise met either way.

Added check to existing newpass so that script could continue when that variable was not provided so the original default one would trigger it erroneously even if not creating a mangos user.

Fixed all mysql calls for version 8 or the way I was calling the program that was having trouble setting default database name to use flag prefix which works better.

Recently also heard of a want to disable all tasks manually first, so added choice for that.

Simplified color check.

---
This change is [Reviewable](https://reviewable.io/reviews/mangosone/database/99)

### CODE REVIEW

**Comment by billy1arm** [comment]
@Pysis868  - I noticed a minor problem with the script when you change the options to no:

Selecting N for Next closes the script - this is wrong !!

Could you also up the version number to 2.1... Thanks

image

File: InstallDatabases.bat [hunk 1]
```diff
@@ -1,1263 +1,1308 @@
...
+if %createMangosUser% == YES set /p newpass=%colYellowBold% New MySQL user password?                   %colReset%DEFAULT: [%colWhiteBold%%newpass%%colReset%] :
+if %newpass%. == . set newpass=mangos
+
+:AfterStep1
+if "%mysqlAuth%" == "" set mysqlAuth=-h %svr% --user=%user% --password=%pass% --port=%port%
+set mysqlString="%mysqlDir:"=%mysql.exe" %mysqlAuth% -q -s
+if "%DEBUG%" == "true" set mysqlString=echo %mysqlString%
+if "%DEBUG%" == "TRUE" set mysqlString=echo %mysqlString%
+
+if %createMangosUser% == YES if %newpass% == mangos set defaultsused=YES
```
Comment (billy1arm) [inline InstallDatabases.bat:498]: Is line 497 correct, the syntax looks a little 'off'

Comment (Pysis868) [reply InstallDatabases.bat:498]: Those are nested terse conditions, omitting 2 pairs of optional parens.

**Comment by billy1arm** [comment]
@Pysis868 would you also be able to add in the changes made to the bat file in this commit:

https://github.com/mangoszero/database/commit/46300f1d5b14a9ff7d27de0b3efdeb3889e59f39

**Comment by billy1arm** [comment]
@Pysis868 sorry, spotted another....
 Line 996 is using the previous style of database load string:

https://github.com/mangosone/database/blob/95b4bdd0e89c91da987e812c87ecb25d54232748/InstallDatabases.bat#L996

**Comment by Pysis868** [comment]
Antz or either of us wants to test running this locally on a Windows PC and loading a remote database.  I think it's debugging that loader error he experienced, will probably include some dlls, maybe a different/updated exe.

BIO_st_callback_ex error

**Comment by billy1arm** [comment]
`@Pysis868` I tried a clean install of this pr this morning and got the following issues:

_______________________________________________________________________________
|    __  __      _  _  ___  ___  ___                                          |
|   |  \/  |__ _| \| |/ __|/ _ \/ __|                                         |
|   | |\/| / _` | .` | (_ | (_) \__ \                                         |
|   |_|  |_\__,_|_|\_|\___|\___/|___/ Database Setup and World Loader         |
|_____________________________________________________________________________|
|                                                                             |
| Only leave the defaults for username, password and mysql port on servers    |
| with no external internet access.                                           |
|                                                                             |
| Using these settings will leave you server open to exploits,                |
| security breaches... or much much worse.                                    |
|_____________________________________________________________________________|


 What is your MySQL host name?           DEFAULT: [localhost] :~~~.~~~.~~~.~~~
 What is your MySQL user name?                DEFAULT: [root] :
 What is your MySQL password?               DEFAULT: [mangos] :~~~~~~~~~~~~~~
 What is your MySQL port?                     DEFAULT: [3306] :
 What is your Character database name?  DEFAULT: [character1] :c1n
 What is your World database name?         DEFAULT: [mangos1] :m1n
 What is your Realm database name?          DEFAULT: [realmd] :r1n
_______________________________________________________________________________
|                                                                             |
| Creating World Database                                                     |
|_____________________________________________________________________________|

 Creating Database m1n
'-e' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Populating World Database                                                   |
|_____________________________________________________________________________|

'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\areatrigger_tavern.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\areatrigger_teleport.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\autobroadcast.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\battleground_events.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\battleground_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\battlemaster_entry.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\command.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\conditions.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_addon.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_ai_scripts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_ai_summons.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_ai_texts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_battleground.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_equip_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_equip_template_raw.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_item_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_linking.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_linking_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_model_info.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_model_race.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_movement.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_movement_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_onkill_reputation.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_template_addon.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_template_classlevelstats.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\creature_template_spells.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\custom_texts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\db_scripts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\db_script_string.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\db_version.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\disables.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\disenchant_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\exploration_basexp.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\fishing_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gameobject.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gameobject_battleground.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gameobject_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gameobject_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event_creature.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event_creature_data.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event_gameobject.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event_mail.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_event_quest.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_graveyard_zone.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_tele.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\game_weather.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gossip_menu.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gossip_menu_option.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\gossip_texts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\instance_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\item_enchantment_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\item_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\item_required_target.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\item_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_creature.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_gameobject.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_gossip_menu_option.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_item.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_npc_text.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_page_text.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_points_of_interest.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\locales_quest.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\mail_level_reward.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\mail_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\mangos_string.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\npc_text.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\npc_trainer.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\npc_trainer_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\npc_vendor.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\npc_vendor_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\page_text.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\petcreateinfo_spell.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pet_levelstats.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pet_name_generation.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pickpocketing_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\playercreateinfo.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\playercreateinfo_action.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\playercreateinfo_item.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\playercreateinfo_spell.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\player_classlevelstats.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\player_levelstats.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\player_xp_for_level.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\points_of_interest.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_creature.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_creature_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_gameobject.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_gameobject_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_pool.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\pool_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\prospecting_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\quest_relations.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\quest_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\reference_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\reputation_reward_rate.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\reputation_spillover_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\reserved_name.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\script_binding.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\script_texts.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\script_waypoint.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\skill_discovery_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\skill_extra_item_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\skill_fishing_base_level.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\skinning_loot_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_affect.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_area.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_bonus_data.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_chain.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_elixir.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_learn_spell.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_linked.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_pet_auras.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_proc_event.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_proc_item_enchant.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_script_target.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_target_position.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_template.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\spell_threat.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\transports.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
. Loading World\Setup\FullDB\warden.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Creating Character Database                                                 |
|_____________________________________________________________________________|

 Creating Database c1n
'-e' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Loading Character Database                                                  |
|_____________________________________________________________________________|

 Loading Database c1n
'-D' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Creating Realm Database                                                     |
|_____________________________________________________________________________|

 Creating Database r1n
'-e' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Loading Realm Database                                                      |
|_____________________________________________________________________________|

 Loading Database r1n
'-D' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
| Adding RealmList entry in Realm Database                                    |
|_____________________________________________________________________________|

 Adding Realmlist entry to r1n
_______________________________________________________________________________
'-D' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
|                                                                             |
| Applying Character DB updates                                               |
|                                                                             |
|_____________________________________________________________________________|
_______________________________________________________________________________
|                                                                             |
|                                                                             |
| Applying World DB updates                                                   |
|                                                                             |
|_____________________________________________________________________________|
World\Updates\Rel21\Rel21_13_003_carriage_returns_quest.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_13_004_Fix_typo_Skuller_to_Skulker.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_14_001_GM_tickets_handling_fixes.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_001_GM_Commands_localization.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_002_minor_mangos_string_update.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_003_The_Big_Command_Help_Sync.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_004_The_Big_Command_Help_Sync_pt2.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_005_DB_Cleanup_pt1.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_006_DB_Cleanup_pt2.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_007_DB_Cleanup_pt3.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_008_DB_Cleanup_pt4.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_009_DB_Cleanup_pt5.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_010_DB_Cleanup_pt6.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_011_DB_Cleanup_pt7.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_012_Illidari_Jailor_21520.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_013_Q_770_Horde_only.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_014_Velendris_Whitemorn.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_015_Add_missing_item_13325.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_016_Fixed_text_for_item_10022.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_017_Add_XT9_pathing.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_018_Laris_Geardawdle.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_019_Felfire_Diemetradon_AI.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_020_Spell_12613_target.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_021_Brainwashed_Noble.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_022_Overlord_Gorefist.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_023_Slaag_22199_AI_fix.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_024_Wrath_Lord.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_025_Improve_Broken_Skeleton_16805.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_026_Skyguard_Rations_Buycount.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_027_Fix_texts_for_quest_6461.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_028_Fix_text_for_Q8288.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_029_Add_missing_spawns_NPC_12125.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_030_Dire_Maul_Spawn_updates_pt1.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_031_Dire_Maul_Spawn_updates_pt2.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_032_Dire_Maul_Spawn_updates_pt3.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
World\Updates\Rel21\Rel21_15_033_.sql
'-D' is not recognized as an internal or external command,
operable program or batch file.
_______________________________________________________________________________
|                                                                             |
|                                                                             |
| Applying Realmd DB updates                                                  |
|                                                                             |
|_____________________________________________________________________________|
_______________________________________________________________________________
|    __  __      _  _  ___  ___  ___                                          |
|   |  \/  |__ _| \| |/ __|/ _ \/ __|                                         |
|   | |\/| / _` | .` | (_ | (_) \__ \                                         |
|   |_|  |_\__,_|_|\_|\___|\___/|___/ Database Setup and World Loader         |
|_____________________________________________________________________________|
|                                                                             |
|                                                                             |
| Database Creation and Load complete                                         |
|                                                                             |
|_____________________________________________________________________________|

 Script has completed !
 - Please review the messages above to ensure no errors occurred.

Press any key to continue . . .`

**Comment by Cobollatin** [comment]
For anyone who has installed mysql 8.x.
Using the mysql binary from tools/ would cause an error.
Just changed it to an empy value(e.i. set mysql= ) and it worked.

**Comment by billy1arm** [comment]
Implemented in:
https://github.com/mangosone/database/commit/5d7b02fcd0f511b46ad0b25c04f9a27e2017cec6

**Pull request closed without merging**